resources/app.py

```diff
@@ -366,6 +390,88 @@ def get_bed_reservations(db=Depends(get_db)):
     return bed_reservations
 
 
+@app.patch("/update/bed_reservation_time/{reservation_id}", tags=["Bed Reservation"])
+def update_bed_reservation_time(
```

Comment:
If i have reservation 2024-04-22 to 2024-04-22 and i want to prolong it for next 3 days, to 2024-04-25, i am getting `"Patient already has a bed reservation for this period"`
Question is:
If we want to prolong reservation should we create new one or enable it to be prolonged in the same reservation?